src/reporters/browser.js

```diff
@@ -38,10 +37,15 @@ export default class BrowserReporter {
     `
 
     // Log to the console
-    consoleLogFn(
-      '%c' + [tag, type].filter(Boolean).join(':'),
-      style,
-      ...logObj.args
-    )
+    if (typeof logObj.args[0] === 'string') {
+      consoleLogFn(
+        `%c${[tag, type].filter(Boolean).join(':')}%c ${logObj.args[0]}`,
+        style,
+        '',
```

Comment:
ℹ️ This resets the style to defaults

Reply:
Why do you think adding a comment above statement about it?